src/components/MoreInformation.js

```diff
@@ -53,12 +55,100 @@ function ExtraData({ data }) {
     } else if (typeof data == "object") {
         let elements = [];
         for (const key in data) {
-            elements.push(
-                <div className={classes.extraData}>
-                    <b>{key}:</b> {data[key]}
-                </div>
-            );
+            //formats additional information instead of just adding it to the more information section
+            if (key === "Additional Information") {
+                const currentData = data[key];
+                if (currentData) {
+                    var finalData = parseMD(currentData);
+                    //adds the parsed data as straight html
+                    elements.push(
+                        <div
+                            className={classes.extraData}
+                            dangerouslySetInnerHTML={{
```

Comment:
Sorry, I might've missed when we decided to use dangerouslySetInnerHTML but I think we should avoid this at all costs. As @MattRC7 mentioned, this is a security risk.